### DISCUSSION TITLE
[Client]这个外层容器指的是div嘛，为啥我填充的时候总是跑到页面最下面

### DISCUSSION THREAD

**Discussion by ruisen2020** [Tencent/cherry-markdown#957] [Q&A]
### Prerequisites

- [ ] There isn't an existing issue that requests the same feature, to avoid duplicates.

### Issue Type

Bug Report

### Description of the Problem or Idea

![image](https://github.com/user-attachments/assets/c6ed46f0-9ce7-4da2-a532-2054375362bb)
这个外层容器指的是div嘛，为啥我填充的时候总是跑到页面最下面

### Contributing

None

**Comment by RSS1102** [comment]
请提供最小复现，以便更好地排查问题

**Reply by ruisen2020** [reply]
这个问题解决了，但是我还有一个问题， new Cherry({
          id: "dialogMarkdown",
          value: this.commentText)  创建实例的时候，这个value能动态绑定嘛，比如说我修改markdown了，commentText也能跟着变嘛

**Reply by RSS1102** [reply]
请使用 :[getMarkdown()](https://tencent.github.io/cherry-markdown/examples/api.html)

**Comment by mikufufucom** [comment]
我也遇到了同样的问题的，请问是怎么解决的呀
